### PR TITLE
Introduce docs for etcd migration

### PR DESCRIPTION
**Pull request by ingvagabund**

### CODE REVIEW

File: admin_guide/etcd_migration.adoc [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+It is highly recommended to perform all the steps automatically.
+If any of the etcd members is started too late, the member will not get accepted by the cluster.
+
+== Deployments
+
+The etcd migration is supported over the following deployments:
+
+* rpm based deployment
+* container based deployment
+
```
Comment (ingvagabund) [inline admin_guide/etcd_migration.adoc]: @sdodson do/will we support etcd migration for AH?

Comment (sdodson) [inline admin_guide/etcd_migration.adoc]: Yeah, we'll need to. We should be able to run this inside the master container. Lets sync up and see if we can get that implemented.

**Comment by sdodson** [comment]
Reading through this, we make no mention of the playbooks. I'll try to find some time to push some commits to the branch to add that but I may not get time. I think this is a case where we should put the playbook front and center and then elaborate on what we're actually doing after that. Making it clear that the playbook handles all of that for you.

**Comment by sdodson** [comment]
Reading through this, we make no mention of the playbooks. I'll try to find some time to push some commits to the branch to add that but I may not get time. I think this is a case where we should put the playbook front and center and then elaborate on what we're actually doing after that. Making it clear that the playbook handles all of that for you.

**Comment by sdodson** [comment]
We're going to hold documenting this process while we test things more extensively.

**Comment by adellape** [comment]
@ingvagabund @sdodson Anything you want to change in here at this point before I start making edits?

**Comment by sdodson** [comment]
Yeah, can I have until the end of day tomorrow to update it? The biggest thing I want to add is how to recover from a failed migration in the unlikely case you have to do so.

**Comment by sdodson** [comment]
Ok, updated to add recovery procedure and to accurately represent the new process.

**Comment by adellape** [comment]
Thanks, moving to https://github.com/openshift/openshift-docs/pull/4652.

**Pull request closed without merging**